frontend/public/components/cluster-service-class.tsx

```diff
@@ -1,16 +1,17 @@
 /* eslint-disable no-undef */
 
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import * as _ from 'lodash-es';
 
+import { Link } from 'react-router-dom';
 import { ColHead, DetailsPage, List, ListHeader, ListPage, ResourceRow } from './factory';
-import { SectionHeading, detailsPage, navFactory, ResourceSummary, resourcePathFromModel } from './utils';
+import {SectionHeading, detailsPage, navFactory, ResourceSummary, resourcePathFromModel, ResourceLink} from './utils';
```

Comment:
style nit: please keep consistent whitespace inside braces like the other lines

Reply:
ugh..sorry, think my ide is auto-formatting these.